pubspec.yaml

```diff
@@ -19,9 +19,9 @@ dependencies:
   mustache4dart: ^1.0.9
   package_config: '>=0.1.5 <2.0.0'
   path: ^1.3.0
-  quiver: '>=0.18.0 <0.25.0'
-  resource: '>=1.1.0 <3.0.0'
+  resource: ^1.1.0
```

Comment:
Not sure.  I just didn't see the reason for the ceiling so removed it (if we can cross one major version, likely we can cross others).

I can bump to ^2.1.0, add the ceiling, or both.  What do you think?